src/Http/Cookie/CookieCollection.php

```diff
@@ -156,15 +155,20 @@ public function get(string $name): CookieInterface
      * @return bool True if the cookie exists, otherwise false.
      */
     public function has(string $name): bool
+    {
+        return null !== $this->__get($name);
+    }
+
+    public function __get(string $name): ?CookieInterface
```

Comment:
If we're going to implement `__get` we should also implement `__isset()`. Having a readable property that can't be checked with isset/empty is clunky.

Reply:
@markstory Please check.